SEED-platform/seed
# Updates to portfolio summary

#### Any background context you want to provide?
The following is based on user feedback.

#### What's this PR do?
- Allows users to sort on portfolio summary columns that represent data from related models (`goal_note` and `historical_note`).
- Fixes bug that ignored 0 values in data quality checks, leading to false "passed checks"
- Excludes baseline new builds from summary calculations

#### How should this be manually tested?
- Attempt to sort on Question, Resolution, Historical Note, and Passed Checks
- Set EUI to 0 for a property, rerun data quality checks. Property should fail and a label applied
- Set New Build or Acquired to true for a property, baseline calculations should update to ignore this property (looking at the change in total area is easiest), while current calculations remain unchanged. 
#### What are the relevant tickets?
#4852
#4583

#### Screenshots (if appropriate)
